app/services/account_oauth_ender.go

```diff
@@ -0,0 +1,47 @@
+package services
+
+import (
+	"math"
+	"strings"
+
+	"github.com/keratin/authn-server/app/data"
+)
+
+func AccountOauthEnder(store data.AccountStore, accountId int, providers []string) error {
```

Comment:
I would suggest `IdentityRemover`.

It's quite likely that all of the "Identity" service functions should have a different noun, but I would request handling that as a separate branch.

Reply:
@cainlevy agree - I can certainly rename these functions in a separate PR. For now, it would be more beneficial to limit the scope of changes to avoid increasing the size of the current PR.

Reply:
Thanks for the suggestions, the function was renamed to `IdentityRemover`.